app/client/src/components/formControls/DropDownControl.tsx

```
@@ -152,6 +155,54 @@ function renderDropdown(
+  // the group. If not present or the type is not defined in the optionGroupConfig then it will be
+  // added to the default group mentioned above.
+  if (
+    !!props.optionGroupConfig &&
+    typeof props.optionGroupConfig === "object"
+  ) {
+    optionGroupConfig = props.optionGroupConfig;
+    options.forEach((opt) => {
+      let optionGroupType = defaultOptionGroupType;
+      let groupConfig: DropDownGroupedOptionsInterface;
+
+      if (opt.hasOwnProperty("optionGroupType") && !!opt.optionGroupType) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)